provider/aws/registries.go

```diff
@@ -13,6 +13,27 @@ import (
 	docker "github.com/fsouza/go-dockerclient"
 )
 
+func isOurOwnECR(p *AWSProvider, server string) bool {
```

Comment:
`host := fmt.Sprintf("%s.dkr.ecr.%s.amazonaws.com", a.Outputs["RegistryId"], p.Region)`

Found this builds.go in the provider. Should be able to use that to match specifically.